specs/subresourceintegrity/spec.markdown

```diff
@@ -534,7 +534,7 @@ value each element's `integrity` content attribute is added to the
 
 A future revision of this specification is likely to include integrity support
 for all possible subresources, i.e., `a`, `audio`, `embed`, `iframe`, `img`,
-`link`, `object`, `script`, `source`, `track`, and `video` elements.
+`object`, `script`, `source`, `track`, and `video` elements.
```

Comment:
there are other types of link tags too and we only do styles currently. So not sure if we should remove it: see, e.g., https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types